tests/envs/test_action_dim_check.py

```diff
@@ -17,3 +19,63 @@ def test_serialize_deserialize(environment_id):
 
     with pytest.raises(ValueError, match="Action dimension mismatch"):
         env.step(0.1)
+
+
+@pytest.mark.parametrize(
+    "environment_id",
```

Comment:
Could you use the `spec_list` and filter for all environments that have a discrete action space?